src/shipit_static_analysis/shipit_static_analysis/clang/tidy.py

```diff
@@ -120,16 +120,14 @@ def parse_issues(self, clang_output, revision):
         Parse clang-tidy output into structured issues
         '''
 
-        # Limit clang output parsing to 'Enabled checks:'
-        end = re.search(r'^Enabled checks:\n', clang_output, re.MULTILINE)
-        if end is not None:
-            clang_output = clang_output[:end.start()-1]
-
         # Sort headers by positions
         headers = sorted(
             REGEX_HEADER.finditer(clang_output),
             key=lambda h: h.start()
         )
+        if not headers:
+            logger.warn('No clang-tidy header was found even though a clang output was provided')
```

Comment:
If this is not supposed to happen, maybe throw, or make the log more severe?

Reply:
I'll leave it to `warn` as it can happen, when there are genuinely no issues described. This should be a concern only if we see this warning on every report though.

Reply:
I found a line at the end of the output `XX warnings present.` where `XX` is an integer (itc an be 0, for valid clang files without issues).
This line is not present when no analysis is made (non-clang files), so we can raise an Exception is no header is found when `XX` is present and positive

Reply:
Sounds good!